src/utils/helpers/DetailsDrawHelpers.ts

```diff
@@ -189,9 +194,15 @@ export function initializeDetailsView(node: TreeNode) {
     const height = Number(detailsViewContainer.attr('height'));
 
     const detailsZoom = selectDetailsZoom();
-
     createDiagrams(detailsZoom, consumerNodes, providerNodes, width, height);
     switchDetailsVisibility();
+
+    const detailsContainerDiv = selectDetailsContainerDiv().node();
+    const { width: containerWidth, height: containerHeight } = detailsContainerDiv
```

Comment:
should't this logic be inside resetZoomPosition ?

Reply:
It should 👍 , gonna change